sections/exchange/FooterCard/TradeSummaryCard/TradeSummaryCard.tsx

```diff
@@ -32,87 +31,88 @@ type TradeSummaryCardProps = {
-	]);
+		onSubmit,
+		feeReclaimPeriodInSeconds,
+		quoteCurrencyKey,
+		showFee = true,
+		totalFeeRate,
+		baseFeeRate,
+		transactionFee,
+		feeCost,
+		isApproved = true,
+		...rest
+	}) => {
+		const { t } = useTranslation();
+		const { useEthGasPriceQuery } = useSynthetixQueries();
 
-	const ethGasPriceQuery = useEthGasPriceQuery();
+		const isSubmissionDisabled = useMemo(() => submissionDisabledReason != null, [
+			submissionDisabledReason,
+		]);
 
-	const gasPrices = useMemo(
-		() => (ethGasPriceQuery.isSuccess ? ethGasPriceQuery?.data ?? undefined : undefined),
-		[ethGasPriceQuery.isSuccess, ethGasPriceQuery.data]
-	);
+		const ethGasPriceQuery = useEthGasPriceQuery();
 
-	const summaryItems = (
-		<SummaryItems attached={attached}>
-			<GasPriceSelect gasPrices={gasPrices} transactionFee={transactionFee} />
-			{showFee && (
-				<>
-					<FeeRateSummaryItem totalFeeRate={totalFeeRate} baseFeeRate={baseFeeRate} />
-					<FeeCostSummaryItem feeCost={feeCost} />
-				</>
-			)}
-		</SummaryItems>
-	);
+		const gasPrices = useMemo(() => ethGasPriceQuery?.data ?? undefined, [ethGasPriceQuery.data]);
 
-	return (
-		<>
-			<MobileOrTabletView>
-				<MobileCard className="trade-summary-card">
-					<Card.Body>{summaryItems}</Card.Body>
-				</MobileCard>
-			</MobileOrTabletView>
-			<MessageContainer attached={attached} className="footer-card" {...rest}>
-				<DesktopOnlyView>{summaryItems}</DesktopOnlyView>
-				<ErrorTooltip
-					visible={feeReclaimPeriodInSeconds > 0}
-					placement="top"
-					content={
-						<div>
-							{t('exchange.errors.fee-reclamation', {
-								waitingPeriod: secondsToTime(feeReclaimPeriodInSeconds),
-								currencyKey: quoteCurrencyKey,
-							})}
-						</div>
-					}
-				>
-					<span>
-						<Button
-							disabled={isSubmissionDisabled}
-							onClick={onSubmit}
-							size="lg"
-							data-testid="submit-order"
-							fullWidth
-						>
-							{isSubmissionDisabled
-								? submissionDisabledReason
-								: !isApproved
-								? t('exchange.summary-info.button.approve')
-								: t('exchange.summary-info.button.submit-order')}
-						</Button>
-					</span>
-				</ErrorTooltip>
-			</MessageContainer>
-		</>
-	);
-};
+		const summaryItems = useMemo(
+			() => (
+				<SummaryItems>
+					<GasPriceSelect gasPrices={gasPrices} transactionFee={transactionFee} />
+					{showFee && (
+						<>
+							<FeeRateSummaryItem totalFeeRate={totalFeeRate} baseFeeRate={baseFeeRate} />
+							<FeeCostSummaryItem feeCost={feeCost} />
+						</>
+					)}
+				</SummaryItems>
+			),
+			[gasPrices, transactionFee, totalFeeRate, baseFeeRate, feeCost, showFee]
+		);
+
+		return (
+			<>
+				<MobileOrTabletView>
+					<MobileCard className="trade-summary-card">
+						<Card.Body>{summaryItems}</Card.Body>
+					</MobileCard>
+				</MobileOrTabletView>
+				<MessageContainer className="footer-card" {...rest}>
+					<DesktopOnlyView>{summaryItems}</DesktopOnlyView>
+					<ErrorTooltip
```

Comment:
Some minor CSS issue with this tooltip, it displays above the modal.

![image](https://user-images.githubusercontent.com/10401554/189984906-568f506c-711f-4cfc-bd44-ca6fb0ef215a.png)

![image](https://user-images.githubusercontent.com/10401554/189984937-99fc6975-dc08-4d03-9ade-625c0a9653ff.png)